src/logic/currentSession/store/reducer/currentSession.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import { Map } from 'immutable'
+import { handleActions, type ActionType } from 'redux-actions'
+import { LOAD_CURRENT_SESSION } from '~/logic/currentSession/store/actions/loadCurrentSession'
+import { UPDATE_VIEWED_SAFES } from '~/logic/currentSession/store/actions/updateViewedSafes'
+import { saveCurrentSessionToStorage } from '~/logic/currentSession/utils'
+
+export const CURRENT_SESSION_REDUCER_ID = 'currentSession'
+
+export type State = Map<string, *>
+
+export default handleActions<State, *>(
+  {
+    [LOAD_CURRENT_SESSION]: (
+      state: State,
+      action: ActionType<Function>,
+    ): State => state.set(CURRENT_SESSION_REDUCER_ID, action.payload),
```

Comment:
why duplicate the reducer id? state now looks like that:
```
state: {
  currentSession: {
     currentSession: {}
  }
}
```
without duplicating the id it would look like that
```
state: {
  currentSession: {}
}
```
the latter seems better imo, wdyt?

Reply:
Yes, it's better. I wasn't able yesterday to do it. But now I got the idea :bulb: and did the changes.

Thanks!